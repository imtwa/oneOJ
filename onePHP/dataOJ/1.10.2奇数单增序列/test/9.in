212
289 127 459 321 106 312 424 88 124 422 385 417 332 403 100 175 261 499 53 167 488 65 440 320 233 156 174 95 79 74 336 208 380 447 124 342 414 412 129 407 350 347 367 134 74 227 136 331 486 421 85 443 359 127 317 460 449 109 197 174 194 243 152 391 481 137 178 408 117 309 174 259 457 322 97 123 225 286 402 228 392 277 231 50 319 105 377 61 431 74 236 448 452 194 160 391 216 384 343 247 94 216 173 318 70 103 100 128 337 203 423 250 140 226 197 391 119 189 290 322 244 174 148 346 296 493 192 269 364 365 203 433 54 278 459 84 317 99 229 342 347 412 388 390 449 323 241 386 348 77 280 274 152 480 315 128 472 458 63 232 167 296 404 184 314 358 432 137 75 483 318 442 66 113 272 173 417 140 312 377 83 93 163 239 325 309 196 363 121 378 494 367 493 100 99 71 153 320 349 184 283 393 196 374 391 382 276 152 153 340 427 364 