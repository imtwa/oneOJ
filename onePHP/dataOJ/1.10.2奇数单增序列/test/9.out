53,61,63,65,71,75,77,79,83,85,93,95,97,99,99,103,105,109,113,117,119,121,123,127,127,129,137,137,153,153,163,167,167,173,173,175,189,197,197,203,203,225,227,229,231,233,239,241,243,247,259,261,269,277,283,289,309,309,315,317,317,319,321,323,325,331,337,343,347,347,349,359,363,365,367,367,377,377,385,391,391,391,391,393,403,407,417,417,421,423,427,431,433,443,447,449,449,457,459,459,481,483,493,493,499
